data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/stream/BinlogEventListener.java

```diff
@@ -101,6 +129,26 @@ public void onEvent(com.github.shyiko.mysql.binlog.event.Event event) {
         }
     }
 
+    public void stopClient() {
+        try {
+            binaryLogClient.disconnect();
```

Comment:
You may want to add a debug/info message when the client is disconnected. This will help during performance testing

Reply:
Added.